### ISSUE TITLE
Hide Supply Library / Labor Manufacturer and Model fields

### ISSUE THREAD

**Issue by fishbaugher** [fishbaugher/WAweb-Issues#71]
These two fields don't make sense on this form and are hidden in v8

<img width="421" alt="Untitled" src="https://github.com/fishbaugher/WAweb-Issues/assets/479175/57dd3748-8848-4c53-a66d-a74805a9cdc6">

Work around is to simply ignore the fields.

**Comment by eckmanwe** [comment]
So is the recommendation to ghost "manufacturer" and "model" on the Supply Library Labor Details Form?

**Comment by fishbaugher** [comment]
Yes, either ghost or hide "manufacturer" and "model" for Supply Library records of "Labor" type.

This is really just for credibility (trying not to be snicker worthy..) of the v10, not function, since the fields can just be ignored.
